.make/default.mk

```diff
@@ -0,0 +1,31 @@
+.PHONY: node-generate-dev-default
+node-generate-dev-default: node-generate-prod
+
+.PHONY: node-generate-prod-default
```

Comment:
can we remove this, it does not do anything

Reply:
it does, this allows the overwrite of steps, check the % at the end, this is a catch all which then triggers the defaults.

that way we can overwrite and don’t have to define empty steps in each Makefile